app/views/users/_user.html.haml

```diff
@@ -12,9 +12,12 @@
       %td= user.age
       %td
         - if current_user.admin?
-          =link_to user_path(user), { method: :delete, data: { :confirm => 'Are you sure?' },  title:'delete' } do
+          = link_to user_path(user), { method: :delete, data: { :confirm => 'Are you sure?' },  title:'delete' } do
             %button.btn.btn-default.index-button
               = fa_icon "trash"
+        = link_to send_regard_index_path(user_id: user, current_user_id: current_user ), { method: :post } do
```

Comment:
We should have access to the current user in a controller, so probably sending current_user is not necessary here.